.github/workflows/pr.yaml

```
@@ -143,7 +152,7 @@ jobs:
       build_type: pull-request
       script: ci/test_wheel_cudf.sh
   wheel-build-cudf-polars:
-    needs: wheel-build-pylibcudf
+    needs: [wheel-build-pylibcudf, wheel-build-cudf]
```

This doesn't need to change, I think cudf is not required for cudf-polars in 24.10:
```suggestion
    needs: wheel-build-pylibcudf
```

---

Ah yes you're right. Think this is the result of a bad conflict resolution + me missing that #16299 switched `cudf_polars` from using `cudf` to using `pylibcudf`.

refs:

* https://github.com/rapidsai/cudf/pull/16299/files#diff-1df81151b24d2e33848b66b79aa5f49b42e2890de3909b4d7a5cc67929ac1fcf
* https://github.com/rapidsai/cudf/pull/16612#issuecomment-2299384811